src/NUnitFramework/framework/Internal/RandomGenerator.cs

```diff
@@ -180,5 +181,55 @@ public T GetEnum<T>()
         }
 
         #endregion
+        
+        #region String & Guids
+        
+        /// <summary>
+        /// Default characters for random functions.
+        /// </summary>
+        /// <remarks>Default characters are the English alphabet (uppercase &amp; lowercase), arabic numerals, and underscore</remarks>
+        public const string DefaultStringChars = "abcdefghijkmnopqrstuvwxyzABCDEFGHJKLMNOPQRSTUVWXYZ0123456789_";        
+        
+        /// <summary>
+        /// Generate a random Guid
+        /// </summary>
+        /// <returns>string GUID</returns>
+        public string CreateRandomGuid()
```

Comment:
Makes more sense to return a Guid, not a string.

Should be named consistently with other RandomGenerator methods, i.e. GetGuid